src/Abstractions/Entities/TaskEntity.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.Entities;
+
+/// <summary>
+/// Represents an invokable entity.
+/// </summary>
+public interface ITaskEntity
+{
+    /// <summary>
+    /// Runs an operation for this entity.
+    /// </summary>
+    /// <param name="operation">The operation to run.</param>
+    /// <returns>The response to the caller, if any.</returns>
+    ValueTask<object?> RunAsync(TaskEntityOperation operation);
```

Comment:
I am unsure if `ValueTask` is appropriate here, or should it be just `object? Run(TaskEntityOperation operation);`?